build_scripts/windows/scripts/promptflow.spec

```diff
@@ -0,0 +1,135 @@
+# -*- mode: python ; coding: utf-8 -*-
+from PyInstaller.utils.hooks import collect_data_files
+from PyInstaller.utils.hooks import copy_metadata
+
+datas = [('../resources/CLI_LICENSE.rtf', '.'), ('../../../src/promptflow/NOTICE.txt', '.')]
+datas += collect_data_files('streamlit')
+datas += copy_metadata('streamlit')
+datas += collect_data_files('streamlit_quill')
+hidden_imports = ['streamlit.runtime.scriptrunner.magic_funcs']
+
+service_hidden_imports = ['win32timezone']
+
+block_cipher = None
+
+pf_a = Analysis(
```

Comment:
Could we add more metas for exe?
![image](https://github.com/microsoft/promptflow/assets/47586720/6a400ea6-aa73-4884-b0cf-a95c814d6889)

Reply:
![image](https://github.com/microsoft/promptflow/assets/26239730/95cdb6f9-e131-4690-8bb3-c81afa6d050e)